.jenkins

```diff
@@ -112,6 +112,7 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
+                        sh './scripts/install-compile-time-perf.sh'
```

Comment:
Shouldn't we do this in the `Dockerfile` instead of cloning for every commit?

Reply:
That would be fine with me.